### ISSUE TITLE
Works fine of latest version 👍

### ISSUE THREAD

**Issue by shockergit** [J2TEAM/idm-trial-reset#15]
Thanks for this solution.
Atleast i can use this for my temporary need and buy the software later and support the developer of idm

**Comment by LordDantes** [comment]
thanks bro works like charm my version is 6.38

**Comment by Dhyfer1** [comment]
@J2TEAMIt is not working with the latest version of IDM (v. 6.39) Please fix it

**Comment by FahadBinHussain** [comment]
Yeah, it is not working with the latest version

**Comment by shockergit** [comment]
> Yeah, it is not working with the latest version
> 
> EDIT: It is working with the Register IDM now button
> ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)

You can use FDM its free

**Comment by Dhyfer1** [comment]
> EDIT: It is working with the Register IDM now button
> ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)
> 

What version of Windows did you test it on to make it work? Windows 10 or Windows 11?

**Comment by shockergit** [comment]
> > EDIT: It is working with the Register IDM now button
> > ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)
> 
> What version of Windows did you test it on to make it work? Windows 10 or Windows 11?

Win 10

**Comment by Dhyfer1** [comment]
> > > EDIT: It is working with the Register IDM now button
> > > ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)
> > 
> > What version of Windows did you test it on to make it work? Windows 10 or Windows 11?
> 
> Win 10

If that is in Windows 10, why is the dialog window have rounded corners? Isn't that unique to Windows 11, or what did I miss?

**Comment by shockergit** [comment]
> > > > EDIT: It is working with the Register IDM now button
> > > > ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)
> > > 
> > > 
> > > What version of Windows did you test it on to make it work? Windows 10 or Windows 11?
> > 
> > 
> > Win 10
> 
> If that is in Windows 10, why is the dialog window have rounded corners? Isn't that unique to Windows 11, or what did I miss?

Yes he maybe using win 11

**Comment by jaum20** [comment]
> 
> 
> Yeah, it is not working with the latest version
> 
> EDIT: It is working with the Register IDM now button
> ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)

I got a error saying that IDM was registered with a fake serial number

**Comment by FahadBinHussain** [comment]
> > Yeah, it is not working with the latest version
> > EDIT: It is working with the Register IDM now button
> > ![image](https://user-images.githubusercontent.com/75292632/130338103-d467b608-b083-4cf2-9280-ccdc8108f17c.png)
> 
> I got a error saying that IDM was registered with a fake serial number:
> ![Sem título](https://user-images.githubusercontent.com/36702273/132112543-18e38f56-f0f9-4e63-bf77-52af9922e45b.png)

Yeah, I also get that annoying message every time. But IDM still works 🙂

**Comment by Tahmid-Hossain-287** [comment]
Let me chime in to say it has been working like a charm.
Thank you!

**Comment by Dhyfer1** [comment]
> Let me chime in to say it has been working like a charm.
> Thank you!

Yes, but it is irritating to see from time to time notices such as: IDM has been registered with a false serial number bla bla bla

**Comment by Tahmid-Hossain-287** [comment]
> > Let me chime in to say it has been working like a charm.
> > Thank you!
> 
> Yes, but it is irritating to see from time to time advices such as: IDM has been registered with a false serial number or the serial number has been blocked bla bla bla 😒

I guess to keep our morality alive😅

**Comment by Tahmid-Hossain-287** [comment]
> > Let me chime in to say it has been working like a charm.
> > Thank you!
> 
> Yes, but it is irritating to see from time to time advices such as: IDM has been registered with a false serial number or the serial number has been blocked bla bla bla 😒

Not gonna lie bro, I feel this now.😣

**Comment by Unili-LiZnCu** [comment]
It's not working with IDM 6.40 build 2

**Comment by Dhyfer1** [comment]
It's not working with 6.40 build 11. This program stopped working a long time ago

**Comment by Dhyfer1** [comment]
It's not working with 6.40 build 11. This program stopped working a long time ago
